test_util/cluster.py

```diff
@@ -454,3 +457,22 @@ def run_integration_tests(cluster, setup=True, **kwargs):
             public_agent_list=[h.private_ip for h in cluster.public_agents],
             provider=cluster.provider,
             **kwargs)
+
+
+def get_apps_tasks_details(cluster, install_app=False):
```

Comment:
Removed these indirections.

I was prepared (/preparing) for running these python tests on the cluster. That was the reason for these indirections.